CompareComplyV1.playground/Pages/Batches.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,69 @@
+
```

Comment:
General comment: Can you add the page name at the top in an h2, e.g.
```
//:## Batches
```

That will be consistent with the other playgrounds.